molecule/testinfra/common/test_platform.py

```diff
@@ -15,9 +15,9 @@ def test_ansible_version(host):
 
 def test_platform(host):
     """
-    SecureDrop requires Ubuntu 20.04 LTS
+    SecureDrop requires Ubuntu 20.04 (focal) or 24.04 (noble)
     """
     assert host.system_info.type == "linux"
     assert host.system_info.distribution == "ubuntu"
-    assert host.system_info.codename == "focal"
-    assert host.system_info.release == "20.04"
+    assert host.system_info.codename in ["focal", "noble"]
+    assert host.system_info.release in ["20.04", "24.04"]
```

Comment:
This looks like it would pass if (codename, release) were ("focal", "24.04"), it might be better to check for the expected tuple.

Reply:
Done.